docs/docsite/rst/windows_winrm.rst

```diff
@@ -657,6 +656,29 @@ for each authentication option. See the section on authentication above for more
     encryption done over TLS. The WinRM payload is still encrypted with TLS
     when run over HTTPS, even if ``ansible_winrm_message_encryption=never``.
 
+IPv6 Addresses
+``````````````
+IPv6 addresses can be used instead of IPv4 addresses or hostnames and is set
```

Comment:
Suggestion: end the sentence after the word "hostnames", and then start a new sentence: "This option is normally set in an inventory."